### PR TITLE
Remove use of future package and refactor to use six

### PR DESCRIPTION
**Pull request by BatsShadow**
This should fix some subtle bugs caused by future aliases

### CODE REVIEW

**Comment by BatsShadow** [comment]
Is this 0.8.1 or 0.9.0?

**Comment by BatsShadow** [comment]
I have confirmed this fixes the mysterious `unspecified` equality check problem in our python-framework.

**Comment by BatsShadow** [comment]
> This should fix some subtle bugs caused by future aliases

I have confirmed this fixes the mysterious `unspecified` equality check problem in our python-framework.

File: src/springfield/fields.py [hunk 1]
```diff
@@ -499,7 +498,7 @@ def _resolve_dotted_name(dotted_name):
         except Exception:
             raise_(
                 ValueError,
-                'Invalid class name for EntityField: %s' % dotted_name,
+                ValueError('Invalid class name for EntityField: %s' % dotted_name),
```
Comment (BatsShadow) [inline src/springfield/fields.py:501]: `future.utils.raise_` worked the other way, but `six.reraise` used here expects the second arg to be an exception.

**Comment by six8** [comment]
This seems like a 0.9

File: src/springfield/fields.py (hunk 1 above)
Comment (six8) [reply src/springfield/fields.py:501]: `raise_from(ValueError(...), e)` would be better for Python 3.

**Comment by six8** [comment]
LGTM. Just need to bump the version. Testing?

**Comment by BatsShadow** [comment]
> LGTM. Just need to bump the version. Testing?

Do you want to merge this and then you handle bumping version? That's how we've done these in the past.

The PR itself shows that tests pass in py27 and py3.x. I have also opened our own Pulse PR depending on this which solves the problem I was originally having (tests pass that did not with Springfield 0.8).

So I think this is good.

**Pull request merged**